### DISCUSSION TITLE
ComboBox detect focus selection change when drop-down is open

### DISCUSSION THREAD

**Discussion by PermanentNovice** [AvaloniaUI/Avalonia#14944] [Q&A]
I have added strings to a ComboBox using the Items.Add("my string") method.

Pressing the spacebar or the enter key on the ComboBox opens the drop-down. Moving up or down the list with the keyboard causes an item to gain focus, however, the item is not selected and the SelectedItemChanged event is not invoked. What event, if any, is triggered when cycling through the items when the drop-down is open? What is the best way to get the index of the focused (but not selected) item?

**Comment by timunie** [comment]
DevTools F12 can show you event chain
